.github/workflows/release.yml

```
@@ -108,12 +108,12 @@ jobs:
           go build -o "$BINARY_NAME" -v
       - name: Release Notes
         run: |
-          touch ".github/RELEASE-TEMPLATE.md" && git log --pretty='format:* %h %s%n  * %an <%ae>' --no-merges >> ".github/RELEASE-TEMPLATE.md"
+          touch ".github/RELEASE-TEMPLATE.md" && git log $(git describe HEAD~ --tags --abbrev=0)..HEAD --pretty='format:* %h %s%n  * %an <%ae>' --no-merges >> ".github/RELEASE-TEMPLATE.md"
```

```suggestion
          touch ".github/RELEASE-TEMPLATE.md" && git log $(git describe HEAD --tags --abbrev=0)..HEAD --pretty='format:* %h %s%n  * %an <%ae>' --no-merges >> ".github/RELEASE-TEMPLATE.md"
```

Should we use `HEAD` instead of `HEAD~`? `HEAD` is where we change version number in constant.go

---

Oh I was wrong. nvm